BP/common/religion/religions/00_celtic.txt

```diff
@@ -180,14 +180,12 @@
 
 	faiths = {
 		celtic_pagan = {
-			color = { 0.6 0.0 0.0 }
-			icon = pagan
-			reformed_icon = pagan_reformed
-
+			color = { 0 0.30 0 }
+			icon = mfi_celtic_pagan
 			doctrine = unreformed_faith_doctrine
 
-			holy_site = paderborn
-			holy_site = hiiumaa
+			holy_site = santiago
+			holy_site = toulouse
```

Comment:
Where do these holy sites come from?

Reply:
Paderborn and Hiiumaa are german and baltic holy sites respectively. Santiago and Toulouse are closer and already shared with other faiths (catholicism/arianism and basque paganism, respectively). I've tried to minimize the number of new holy sites I've made. 

Alternatively, I can work to map the holy sites directly from WtWSMS, but I'll need to figure out effects for all the new ones.

Reply:
We should map holy sites directly, I can make a list of themes for effects if necessary.

Reply:
Themes are the biggest impediment. Otherwise I can map these to the map simply.

Reply:
Ok, I will note that them and see what I can list.